File: src/main/java/com/dh/ondot/member/core/exception/TokenExpiredException.java
package com.dh.ondot.member.core.exception;

import com.dh.ondot.core.exception.UnauthorizedException;

import static com.dh.ondot.core.exception.ErrorCode.TOKEN_EXPIRED;

public class TokenExpiredException extends UnauthorizedException {

    public TokenExpiredException() {
        super(TOKEN_EXPIRED.getMessage());
    }

    @Override
    public String getErrorCode() {
        return TOKEN_EXPIRED.name();
    }
}

File: src/main/kotlin/com/dh/ondot/member/core/exception/TokenExpiredException.kt
package com.dh.ondot.member.core.exception

import com.dh.ondot.core.exception.ErrorCode.TOKEN_EXPIRED
import com.dh.ondot.core.exception.UnauthorizedException

class TokenExpiredException :
    UnauthorizedException(TOKEN_EXPIRED.message) {
    override val errorCode: String get() = TOKEN_EXPIRED.name
}
